src/Module.Server/ModuleData/dtv/dtv_data.xml

```diff
@@ -339,18 +339,16 @@
     <Wave>
       <Group classDivisionId="crpg_dtv_greyorder_shielder" count="1.5" />
       <Group classDivisionId="crpg_dtv_greyorder_kuyak_two_handed" count="0.4" />
-      <Group classDivisionId="crpg_dtv_greyorder_balanced_infantry" count="0.8" />
-      <Group classDivisionId="crpg_dtv_greyorder_heavy_two_handed" count="0.2" />
-      <Group classDivisionId="crpg_dtv_greyorder_heavy_polearm" count="0.2" />
+      <Group classDivisionId="crpg_dtv_greyorder_balanced_infantry" count="1.0" />
       <Group classDivisionId="crpg_dtv_greyorder_lieutenant_varadin" />
```

Comment:
Currently you have 2 lieutenants for wave 1-2 and bosses for wave 3. Cool idea, but they are on par with later round lieutenants/bosses. Either make it 1 strong or 2 lesser versions of a lieutenant/boss so they fall more in line with the other special units (dwarf/tincan).

Especially with the coming changes of ballista damage being heavily reduced to special units..

Reply:
I didn't realize a Tincan soldier with 30/18 only has 7 PS.
But then again, GO round is supposed to be a melee-powerhouse round - since they're gonna be much slower next patch, their main weakness is getting kited.

Droob also mentioned Tincans and Dwarves being too weak right now. With Dwarves main strength being ranged and Tincans having cav and also a mix of everything, I'd like to have GO's strength to be very strong melee-only bots.

Would wait how the speed nerf turns out to be until I'd lower their damage output.

As for the lieutenants and bosses, they're fine as they are imo - same thing need to see how big their impact actually is if not insta-killed by ballistas.